src/Annotations/Get.php

```diff
@@ -9,6 +9,7 @@
 /**
  * @Annotation
  */
+#[\AllowDynamicProperties]
```

Comment:
This seems unrelated to the warnings documented?

Reply:
Without `AllowDynamicProperties` attribute, [This code](https://github.com/zircote/swagger-php/blob/2.x/src/Annotations/AbstractAnnotation.php#L140) causes a deprecate error. I thought I did not need this assignment (`$this->$property = $value;`), but I gave the attribute to minimize the changes for the behavior unchanged.

If I can get rid of this code, I don't need the `AllowDynamicProperties` attribute. (Is this unexpected property?) What should I do?

Reply:
Interesting. I have to admit that seem not to be able to reproduce the warnings; that would make everything a bit easier...

Reply:
I am also confused. I suspect my environment and have had others try, but the error is still there. (I would expect the error to occur since the code is dealing with a dynamic propertyn in PHP 8.1.) However, the GH action I performed in another PR did not reproduce the deprecated error...

Reply:
:) I think you have custom error logging settings. I would have thought this should be default for phpunit :shrug: 
```
php -d error_reporting=-1 ./bin/phpunit
```

... on it.

Reply:
Its this test: https://github.com/zircote/swagger-php/blob/2.x/tests/AbstractAnnotationTest.php#L19

Doesn't exist on master any more, otherwise we'd see the same there...

Not quite sure what to do - technically we should probably change the code to throw an exception rather than logging and moving on; just not sure if that is too much of a BC break.
OTOH, this is the first time I've seen this issue, so I would suspect there are not a lot of cases of this happeing out in the wild. On top of that thoses properties are either ignored during serialization or bu any other OpenAPI tools.

I would lean towards not allowing invalid specs to be created, so throwing an exception might be the way forward.

Reply:
I agree with you that it is correct to throw an exception as you think. But in any case, is there any need to set invalid properties? (I pushed a [new commit](https://github.com/zircote/swagger-php/pull/1474/commits/46501392f2870af6b8b02f966c75dd8f8f2b1fbd))

Reply:
Could we remove this again? When not setting the property this  should not be needed.